extensions/git/src/main.ts

```diff
@@ -40,6 +41,11 @@ async function createModel(context: ExtensionContext, outputChannel: OutputChann
 	const model = new Model(git, context.globalState, outputChannel);
 	disposables.push(model);
 
+	commands.registerCommand('git.editor', () => setVsCodeAsGitEditor(git, outputChannel));
+	commands.registerCommand('git.diffTool', () => setVsCodeAsGitDiffTool(git, outputChannel));
+	commands.registerCommand('git.mergeTool', () => setVsCodeAsGitMergeTool(git, outputChannel));
+	commands.registerCommand('git.gitTools', () => setVsCodeAsGitTools(git, outputChannel));
```

Comment:
Please check the rest of the extension's source code, especially `commands.ts`, on how to add commands to this extension.